pkg/routeagent_driver/handlers/ovn/connection.go

```diff
@@ -122,27 +105,32 @@ func (c *ConnectionHandler) createLibovsdbClient(dbAddress string, tlsConfig *tl
 		libovsdbclient.WithLogger(&logger.Logger),
 	}
 
-	if strings.HasPrefix(dbAddress, "IC:") {
-		node, err := nodeutil.GetLocalNode(c.k8sClientset)
-		if err != nil {
-			return nil, errors.Wrap(err, "error getting the node")
-		}
+	localNode, err := node.GetLocalNode(c.k8sClientset)
+	if err != nil {
+		return nil, errors.Wrap(err, "error getting the node")
+	}
 
-		annotations := node.GetAnnotations()
+	annotations := localNode.GetAnnotations()
 
-		zoneName, ok := annotations[constants.OvnZoneAnnotation]
-		if !ok {
-			return nil, errors.Wrapf(err, "node %q is missing the %q "+
-				"annotation", node.Name, constants.OvnZoneAnnotation)
-		}
+	zoneName, ok := annotations[constants.OvnZoneAnnotation]
```

Comment:
If the constants are used only by route-agent, can we move them from the constants package.

i.e., 
https://github.com/submariner-io/submariner/blob/devel/pkg/routeagent_driver/constants/constants.go#L57

Reply:
moved to constants inside ovn package.